src/providers/deepinfra/index.ts

```diff
@@ -0,0 +1,14 @@
+import { ProviderConfigs } from "../types";
+import DeepInfraApiConfig from "./api";
+import {DeepInfraChatCompleteConfig, DeepInfraChatCompleteStreamChunkTransform} from "./chatComplete";
+
+const DeepInfraConfig: ProviderConfigs = {
+    chatComplete: DeepInfraChatCompleteConfig,
+    api: DeepInfraApiConfig,
+    responseTransforms: {
+        chatComplete: DeepInfraChatCompleteConfig,
```

Comment:
Looks like this is incorrect. The transform function needs to be used here instead of the Config object. Can you please fix this.